src/main/webapp/app/lecture/overview/course-lectures/attachment-video-unit/attachment-video-unit.component.html

```
@@ -43,55 +45,97 @@
+                        <jhi-video-player [videoUrl]="playlistUrl()!" [transcriptSegments]="transcriptSegments()" [initialTimestamp]="targetTimestamp()" />
+                    </div>
+                } @else if (isFullscreen()) {
+                    <div #videoContainer class="video-player-container video-player-container--fullscreen">
+                        <div class="video-iframe-wrapper video-iframe-wrapper--fullscreen">
+                            <iframe id="videoFrame" class="video-iframe" [src]="videoUrl() | safeResourceUrl" allow="fullscreen"></iframe>
+                        </div>
+                    </div>
+                } @else {
+                    <div #videoContainer class="video-player-container">
+                        <div class="ratio ratio-16x9">
+                            <iframe id="videoFrame" class="rounded w-100 h-100 border-0" [src]="videoUrl() | safeResourceUrl" allow="fullscreen"></iframe>
```

HTMLHint (1.9.2)

[error] The id value [ videoFrame ] must be unique.

(id-unique)